members/badges/index.html

```diff
@@ -13,18 +13,35 @@ <h2 id="get-recognized">Get Recognized</h2>
   badges:
 </p>
 
-{% assign odd_num_of_badges = site.data.badges.size | modulo:2 %}
 {% for badge in site.data.badges %}
-{% cycle '<div class="row">','' %}
+<div class="row badgeset">
   <div class="span4">
-    <div class="badgeholder {{ badge.first | map: 'type' }}">
-      <div id="{{ badge.first.first }}"><span class="icon-{{ badge.first | map: 'icon' }}"></span></div>
-      <div>{{ badge.first | map: 'text' }}</div>
+    <div class="badgeholder {{ badge.type }}">
+      <div id="{{ badge.id }}"><span class="icon-{{ badge.icon }}"></span></div>
+      <div>{{ badge.text }}</div>
     </div>
-    <p>{{ badge.first | map: 'description' }}</p>
+    <p>{{ badge.description }}</p>
+  </div>
+
+  <div class="span4">
+    {% for person in site.categories.members %}
+    {% assign user_roles = person.roles %}
+    {% assign user_username = person.username %}
+
+    {% include badges_load_dynamic_roles_recognition.html %}
+
+    {% if user_roles contains badge.id %}
+    {% if person.img %}
+    <a href="{{person.permalink}}">
```

Comment:
This is where we would put a `title` attribute with the name of the project (if applicable), as @rgrp suggested. Would that be hard?